docs/advanced/mixin/5.overwrite.md

```diff
@@ -0,0 +1,53 @@
+@Overwrite
+=========
+:::caution
+This is ```@Overwrite``` but NOT ```@Override```.
+:::
+
+Overwriting is a technology that discards all codes in a vanilla method and replaces it with your own code. It is a thorough way to change the vanilla mechanisms.
+
+:::danger
+It can be dangerous to completely change vanilla mechanisms.
+:::
+
+You should not consider overwriting unless:
+
+- You want to completely change vanilla mechanisms.
+- You want to remove some codes from these vanilla methods.
+- You want to fix the bugs in these vanilla methods.
+- You have a better implementation than these vanilla codes.
```

Comment:
None of these are potentially good reasons. Overwrites should be used if there is no other potential alternative. You can do most of these things with injections in 90% of cases.